docker/main/ngen-calibration/entrypoint.sh

```diff
@@ -83,12 +127,9 @@ load_object_store_keys_from_docker_secrets() {
 start_calibration() {
     # Start ngen calibration
     echo "$(print_date) Starting serial ngen calibration"
-    # CALIBRATION_CONFIG_FILE=${CALIBRATION_CONFIG_DATASET_DIR}/$(basename $(find ${CALIBRATION_CONFIG_DATASET_DIR} -name "*.yaml" -maxdepth 1 | head -1))
 
-    # TODO: move this to CALIBRATION_CONFIG_DATASET_DIR
-    # NOTE: assumes that calibration dataset will be in realization config dataset AND that it is
-    # the only yaml file at the top level of that dataset.
-    CALIBRATION_CONFIG_FILE=${REALIZATION_CONFIG_DATASET_DIR}/$(basename $(find ${REALIZATION_CONFIG_DATASET_DIR} -name "*.yaml" -maxdepth 1 | head -1))
+    # Find and use copy of config in output dataset
+    CALIBRATION_CONFIG_FILE=$(find ${OUTPUT_DATASET_DIR} -name "*.yaml" -maxdepth 1 | head -1)
```

Comment:
Yaml is a little annoying in that both `yml` and `yaml` are valid and commonly used suffixes. We probably want this instead:

```suggestion
    CALIBRATION_CONFIG_FILE=$(find ${OUTPUT_DATASET_DIR} -type f -iname "*.yaml" -o -iname "*.yml" -maxdepth 1 | head -1)
```